cmd/demo/node.go

```diff
@@ -258,14 +267,62 @@ func (n *node) listen() error {
 	if err != nil {
 		return errors.WithMessage(err, "could not start tcp listener")
 	}
-	go n.client.HandleChannelProposals(n)
+	// Set up persistence.
+	db, err := leveldb.LoadDatabase(config.DatabasePath)
+	if err != nil {
+		return errors.WithMessage(err, "creating/loading database")
+	}
+	persister, err := keyvalue.NewPersistRestorer(db)
+	if err != nil {
+		return errors.WithMessage(err, "creating PersistRestorer")
+	}
+	n.client.EnablePersistence(persister)
```

Comment:
If
- no path given in config or cmd line flag or
- flag `--no-persistence` is set
then don't enable persistence.

Reply:
I will do something like `--use-persistence=(on|off)` so that the option can be overridden, otherwise we would also need a `--yes-persistence`.

Reply:
I see what you mean. Then maybe the cleanest way is to have
- `--no-persistence` -> disables persistence
- `--persistence` -> enables persistence, reading path from config
- `--persistence <dbPath>` -> optional argument overrides config path and enables persistence.

or use your format `--use-persistence=(on|off|dbPath)` following the logic above. I think either is fine, but it should be consistent with the format that the other command line flags use.

Reply:
I used the default from `cobra` now, which means:  
- no option given or `--persistence=false`: Disabled  
- `--persistence` or `--persistence=true`: Enabled
  
Should i still add an argument for the path?